tests/e2e/e2e_test.go

```diff
@@ -521,16 +521,6 @@ func (s *IntegrationTestSuite) TestArithmeticTWAP() {
 	osmoassert.DecApproxEq(s.T(), twapAfterSwapBeforePruning10MsBC, twapFromAfterToNowBC, sdk.NewDecWithPrec(2, 3))
 	osmoassert.DecApproxEq(s.T(), twapAfterSwapBeforePruning10MsCA, twapFromAfterToNowCA, sdk.NewDecWithPrec(2, 3))
 
-	if !s.skipUpgrade {
-		// TODO: we should reduce the pruning time in the v11
-		// genesis to make this test run faster
-		// Currenty, we are testing the upgrade from v11 to v12,
-		// the pruning time is set to whatever is in the upgrade
-		// handler (two days). Therefore, we cannot reasonably
-		// test twap pruning post-upgrade.
-		s.T().Skip("skipping TWAP Pruning test. This can be re-enabled post v12")
-	}
```

Comment:
Doesn't this still need to be updated in the old version genesis?

Reply:
I think so. This skipUpgrade exist when upgrading from v11 to v12, but now it's no longer necessary. Should we delete them?

Reply:
This was needed to be updated for v11 genesis. However, this should already be present in v14:
https://github.com/osmosis-labs/osmosis/blob/v14.x/tests/e2e/initialization/config.go#L361